tox.ini

```diff
@@ -29,27 +31,31 @@ deps =
     wagtail62: wagtail>=6.2,<6.3
     wagtailmain: git+https://github.com/wagtail/wagtail.git@main#egg=Wagtail
 
-install_command = pip install -U {opts} {packages}
+install_command = pip install -e ".[testing]" -U {opts} {packages}
 commands =
     coverage run --source="{toxinidir}/wagtail_storages" -m django test wagtail_storages
     django-admin check
     django-admin makemigrations --check --noinput
     coverage report -m --omit="{toxinidir}/wagtail_storages/tests/*" --fail-under=80
-extras = testing
```

Comment:
https://tox.wiki/en/4.18.1/config.html#extras

this is equivallent to `pip install -e ".[testing]"` for the install command